website/docs/best-practices/how-we-build-our-metrics/semantic-layer-2-setup.md

```diff
@@ -13,9 +13,23 @@ git clone git@github.com:dbt-labs/jaffle-sl-template.git
 cd path/to/project
 ```
 
-Next, before you start writing code, you need to install MetricFlow as an extension of a dbt adapter from PyPI  (dbt Core users only). The MetricFlow is compatible with Python versions 3.8 through 3.11.
+Next, before you start writing code, you need to install MetricFlow:
 
-We'll use pip to install MetricFlow and our dbt adapter:
+<Tabs>
+
+<TabItem value="cloud" label="dbt Cloud">
+
+- [dbt Cloud CLI](/docs/cloud/cloud-cli-installation) &mdash; MetricFlow commands are embedded in the dbt Cloud CLI. This means you can immediately run them once you install the dbt Cloud CLI. Using dbt Cloud means you won't need to manage versioning — your dbt Cloud account will automatically manage the versioning.
```

Comment:
Had back-to-back "this means" sentences. We could trim it a bit
```suggestion
- [dbt Cloud CLI](/docs/cloud/cloud-cli-installation) &mdash; MetricFlow commands are embedded in the dbt Cloud CLI. You can immediately run them once you install the dbt Cloud CLI. Using dbt Cloud means you won't need to manage versioning — your dbt Cloud account will automatically manage the versioning.
```

Reply:
what do you mean? we can never have too much 'means' 🥲